po/fr.po

```diff
@@ -1,5 +1,723 @@
+msgid "Autoscale Corners"
+msgstr "Mise à l'échelle automatique des coins"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:208
+msgid "Uniform Corners"
+msgstr "Uniformiser les bordures"
+
+#: src/Layouts/Partials/BordersPanel.vala:53
+msgid "Borders"
+msgstr "Bordures"
+
+#: src/Layouts/Partials/BordersPanel.vala:64
+msgid "Add border"
+msgstr "Ajouter une bordure"
+
+#: src/Layouts/Partials/FillItem.vala:114
+msgid "Choose fill color"
+msgstr "Choisir la couleur de remplissage"
+
+#: src/Layouts/Partials/FillItem.vala:193
+msgid "Remove fill color"
+msgstr "Supprimer la couleur de remplissage"
+
+#: src/Layouts/Partials/FillItem.vala:277
+msgid "Show fill color"
+msgstr "Afficher la couleur de remplissage"
+
+#: src/Layouts/Partials/FillItem.vala:281
+msgid "Hide fill color"
+msgstr "Masquer la couleur de remplissage"
+
+#: src/Layouts/Partials/FillsPanel.vala:54
+msgid "Fills"
+msgstr "Remplissages"
+
+#: src/Layouts/Partials/FillsPanel.vala:65
+msgid "Add fill color"
+msgstr "Ajouter une couleur de remplissage"
+
+#: src/Layouts/Partials/TransformPanel.vala:96
+msgid "X"
+msgstr "X"
+
+#: src/Layouts/Partials/TransformPanel.vala:96
+msgid "Horizontal position"
+msgstr "Position horizontale"
+
+#: src/Layouts/Partials/TransformPanel.vala:98
+msgid "Y"
+msgstr "Y"
+
+#: src/Layouts/Partials/TransformPanel.vala:98
+msgid "Vertical position"
+msgstr "Position verticale"
+
+#: src/Layouts/Partials/TransformPanel.vala:100
+msgid "W"
+msgstr "W"
+
+#: src/Layouts/Partials/TransformPanel.vala:100
+msgid "Width"
+msgstr "Largeur"
+
+#: src/Layouts/Partials/TransformPanel.vala:102
+msgid "H"
+msgstr "H"
+
+#: src/Layouts/Partials/TransformPanel.vala:102
+msgid "Height"
+msgstr "Hauteur"
+
+#: src/Layouts/Partials/TransformPanel.vala:107
+msgid "Lock Ratio"
+msgstr "Ratio de verrouillage"
+
+#: src/Layouts/Partials/TransformPanel.vala:114
+msgid "R"
+msgstr "R"
+
+#: src/Layouts/Partials/TransformPanel.vala:114
+msgid "Rotation degrees"
+msgstr "Degrés de rotation"
+
+#: src/Layouts/Partials/TransformPanel.vala:126
+msgid "Flip Horizontally"
+msgstr "Retournement horizontal"
+
+#: src/Layouts/Partials/TransformPanel.vala:137
+msgid "Flip Vertically"
+msgstr "Retournement vertical"
+
+#: src/Layouts/Partials/TransformPanel.vala:165
+msgid "Position"
+msgstr "Position"
+
+#: src/Layouts/Partials/TransformPanel.vala:169
+msgid "Size"
+msgstr "Taille"
+
+#: src/Layouts/Partials/TransformPanel.vala:178
+msgid "Opacity"
+msgstr "Opacité"
+
+#: src/Layouts/RightSideBar.vala:111
+msgid "Search Layer"
+msgstr "Rechercher un calque"
+
+#: src/Lib/Managers/ExportManager.vala:192
+#: src/Lib/Managers/ExportManager.vala:219
+msgid "Generating preview, please wait…"
+msgstr "Génération de l'aperçu, veuillez patienter…"
+
+#: src/Lib/Managers/ExportManager.vala:313
+#, c-format
+msgid "Untitled %i"
+msgstr "Sans titre %i"
+
+#: src/Lib/Managers/ExportManager.vala:484
+msgid "Exporting images…"
+msgstr "Exporter des images…"
```

Comment:
This should be
> Export des images en cours...

Reply:
Isn't "Exportation" better than "Export" to describe an action in progress?

Reply:
I never seen the word `exportation` for this usage, only to describe international trade (https://fr.wikipedia.org/wiki/Exportation, https://crisco2.unicaen.fr/des/synonymes/exportation, https://www.cnrtl.fr/definition/exportation, https://www.larousse.fr/dictionnaires/francais/exportation/32306)